vuepress/docs/v6.3.2/docs/releases/README.md

```diff
@@ -100,5 +100,8 @@
   - There is  a known issue with the current nginx ingress that can be worked around by using the `singleHost` option for the configuration of your Entando Applications and binding TLS to that single host path
     - The issue will be solved in the next minor or patch release (6.4.0 or 6.3.3)
 
+### Custom Resources
+  - The EntandoDatabaseService CRD definition needs to be updated for some configurations - spec.database needs to be renamed to spec.databaseName to support external databases.      
```

Comment:
Minor change, sorry didn't think of it earlier - if you add backticks on spec.database and spec.databaseName it will display those as code and match what we do elsewhere.